### ISSUE TITLE
[BUG] My downloads don't start

### ISSUE THREAD

**Issue by DaviMoraesGoulart** [hydralauncher/hydra#656]
### Describe the bug

My downloads of certain games, usually from DODI-REPACK, do not start and remain at 0% all the time, the peers and seeds are also at 0, and there's no point in waiting, it simply won't charge, I can leave it for 5 hours and it won't

### Steps to Reproduce

please solve

### Expected behavior

please solve

### Screenshots

![image](https://github.com/hydralauncher/hydra/assets/172142943/b6d965a9-fcd5-4d5a-8317-ab769b62f029)

### Operating System

Windows 10

### Hydra Version

2.0.1

**Comment by EznoShurek** [comment]
mesmo erro aqui. No yt a galera também!!! precisamos de uma correção!!!!!!!!!

**Comment by zamitto** [comment]
@DaviMoraesGoulart, @EznoShurek 

Could you attach your logs? They are located in  "%appdata/hydra/logs"
We are already working on a fix, but this is not a simple feature to change, in the mean time, I'm trying to find some quick fix. Your logs might have something that helps us

**Comment by luizilha** [comment]
i get the same error where...
The log error below.

[2024-06-23 22:48:09.347] [error] (main) user is not logged in
[2024-06-23 22:48:09.383] [error]        Unhandled rejection Error: user is not logged in
    at HydraApi.revalidateAccessTokenIfExpired (file:///tmp/.mount_hydralseqap3/resources/app.asar/out/main/index.js:932:13)
    at HydraApi.post (file:///tmp/.mount_hydralseqap3/resources/app.asar/out/main/index.js:982:16)
    at createGame (file:///tmp/.mount_hydralseqap3/resources/app.asar/out/main/main-BxyCXgQh.js:64:21)
    at startGameDownload (file:///tmp/.mount_hydralseqap3/resources/app.asar/out/main/index-Cbv99xph.js:789:5)
    at async WebContents.<anonymous> (node:electron/js2c/browser_init:2:82807)
[2024-06-23 22:48:22.518] [error] (main) user is not logged in
[2024-06-23 22:48:22.556] [error]        Unhandled rejection Error: user is not logged in
    at HydraApi.revalidateAccessTokenIfExpired (file:///tmp/.mount_hydralseqap3/resources/app.asar/out/main/index.js:932:13)
    at HydraApi.post (file:///tmp/.mount_hydralseqap3/resources/app.asar/out/main/index.js:982:16)
    at createGame (file:///tmp/.mount_hydralseqap3/resources/app.asar/out/main/main-BxyCXgQh.js:64:21)
    at startGameDownload (file:///tmp/.mount_hydralseqap3/resources/app.asar/out/main/index-Cbv99xph.js:789:5)
    at async WebContents.<anonymous> (node:electron/js2c/browser_init:2:82807)
[2024-06-23 22:49:54.570] [error] (main) user is not logged in
[2024-06-23 22:49:54.605] [error]        Unhandled rejection Error: user is not logged in
    at HydraApi.revalidateAccessTokenIfExpired (file:///tmp/.mount_hydralseqap3/resources/app.asar/out/main/index.js:932:13)
    at HydraApi.post (file:///tmp/.mount_hydralseqap3/resources/app.asar/out/main/index.js:982:16)
    at createGame (file:///tmp/.mount_hydralseqap3/resources/app.asar/out/main/main-BxyCXgQh.js:64:21)
    at startGameDownload (file:///tmp/.mount_hydralseqap3/resources/app.asar/out/main/index-Cbv99xph.js:789:5)
    at async WebContents.<anonymous> (node:electron/js2c/browser_init:2:82807)
[2024-06-23 23:54:37.454] [error] (main) user is not logged in
[2024-06-23 23:54:37.476] [error]        Unhandled rejection Error: user is not logged in
    at HydraApi.revalidateAccessTokenIfExpired (file:///tmp/.mount_hydralAWKb9E/resources/app.asar/out/main/index.js:932:13)
    at HydraApi.post (file:///tmp/.mount_hydralAWKb9E/resources/app.asar/out/main/index.js:982:16)
    at createGame (file:///tmp/.mount_hydralAWKb9E/resources/app.asar/out/main/main-BxyCXgQh.js:64:21)
    at startGameDownload (file:///tmp/.mount_hydralAWKb9E/resources/app.asar/out/main/index-Cbv99xph.js:789:5)
    at async WebContents.<anonymous> (node:electron/js2c/browser_init:2:82807)
[2024-06-23 23:58:27.655] [error]        Unhandled rejection TypeError: fetch failed
    at node:internal/deps/undici/undici:12345:11
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Aria2.http (/tmp/.mount_hydralsvT6Mg/resources/app.asar/node_modules/aria2/lib/JSONRPCClient.js:51:22)
    at async Aria2.call (/tmp/.mount_hydralsvT6Mg/resources/app.asar/node_modules/aria2/lib/JSONRPCClient.js:102:5)
    at async DownloadManager.startDownload (file:///tmp/.mount_hydralsvT6Mg/resources/app.asar/out/main/index.js:1514:24)
[2024-06-24 00:10:57.384] [error] (main) user is not logged in
[2024-06-24 00:10:57.423] [error]        Unhandled rejection Error: user is not logged in
    at HydraApi.revalidateAccessTokenIfExpired (file:///tmp/.mount_hydralsvT6Mg/resources/app.asar/out/main/index.js:932:13)
    at HydraApi.post (file:///tmp/.mount_hydralsvT6Mg/resources/app.asar/out/main/index.js:982:16)
    at createGame (file:///tmp/.mount_hydralsvT6Mg/resources/app.asar/out/main/main-BxyCXgQh.js:64:21)
    at startGameDownload (file:///tmp/.mount_hydralsvT6Mg/resources/app.asar/out/main/index-Cbv99xph.js:789:5)
    at async WebContents.<anonymous> (node:electron/js2c/browser_init:2:82807)

**Comment by leomachadods** [comment]
From what I could see in my logs, the problem was with aria2c, as shown in the print (info.txt)

![Screenshot from 2024-06-24 00-00-42](https://github.com/hydralauncher/hydra/assets/99626076/6599c947-e86c-4939-a83f-3d8472e8f822)

OS: Ubuntu 24.04

After giving the command `sudo apt install aria2`, and then I killed the process in the system monitor and downloaded a different game. And magically it was solved!

**Comment by OtavioSC** [comment]
same error here.
platform: windows 11

logs:

```
[2024-06-25 10:21:07.044] [info]  (main) Failed to connect to aria2, retrying...
[2024-06-25 10:21:07.046] [info]  (main) Connected to aria2
[2024-06-25 10:21:07.316] [info]  (main)  ---- RESPONSE -----
[2024-06-25 10:21:07.316] [info]  (main) 201 post /games {
  id: 'wgD4ZAgZ',
  objectId: '268910',
  lastTimePlayed: null,
  playTimeInSeconds: 0,
  shop: 'steam',
  updatedAt: '2024-06-22T23:33:28.666Z',
  createdAt: '2024-06-22T23:33:28.666Z'
}
```

**Comment by JackEnx** [comment]
fixed on #722, if the problem persist, please report it
